docker-compose/docker-compose.yaml

```diff
@@ -104,12 +106,26 @@ services:
       - tlsServerCAPath=/tlsServerCA.pem
       - tlsServerCSRParametersPath=/resources/tlsServerCSRParameters.json
       - tlsServerPvtKeyPath=/tlsServerPvtKey.key
+      - VAULT_ENDPOINT=http://vault-dev:8233
+      - VAULT_PKI_BASE_DOMAIN=example.com
+      - VAULT_AUTH_METHOD=APP_ROLE
+      - VAULT_ROLE_ID_FILE=/vault/role-id
+      - VAULT_ROLE_SECRET_ID_FILE=/vault/secret-id
+      # - VAULT_MOUNT_PKI=pki
```

Comment:
Can we add some comments as to why these are commented out.

I.e. uncomment them if you need xyz

Reply:
Hi @mdebarros , I don't know about these values. I just copied from a sample configuration provided in the service. I couldn't find any documentation for these params. 

I removed the commented lines for now to make it clear and less confusion.